lib/components/DynamicFonts/FontDownloader.ts

```diff
@@ -97,26 +97,38 @@ export default class FontDownloader {
     fontName: string,
     fontExtension: FontExtension,
     timeout: number): Promise<string> {
-    const {downloadErrorMessage} = this.props;
+    const {downloadErrorMessage, debug} = this.props;
     await this.createPrivateFolderIfNeeded();
     const fileName = this.getFileName(fontName, fontExtension);
     const downloadLocation = this.getPrivateFilePath(fileName);
 
     try {
+      if (debug) {
+        console.log(fontName, 'Starting to download');
+      }
       const result = await this.fs.downloadFile(this.getDownloadFontOptions(fontUri, downloadLocation, timeout))
         .promise;
       if (result.statusCode === 200) {
+        if (debug) {
```

Comment:
We can't use logService here, to log only on DEV? I would add a simple console.log there to use here

Reply:
It might be possible but I don't like it, it will open a precedent of using `src` in the `lib` which can cause cycles.
Adding all logs is a little spammy IMO.